widgets/src/components/TaskCompletionForm/TaskCompletionFormContainer.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProvider } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+import theme from 'theme';
+import CustomEventContext from 'components/TaskDetails/CustomEventContext';
+import WidgetBox from 'components/common/WidgetBox';
+import CompletionForm from 'components/TaskCompletionForm/CompletionForm';
+import { getTask, getTaskForm } from 'api/app-builder/taskDetails';
+import { getPageWidget } from 'api/app-builder/pages';
+
+class TaskCompletionFormContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      config: null,
+      loading: false,
+      formSchema: null,
+      formData: {},
+    };
+
+    this.fetchTaskFormData = this.fetchTaskFormData.bind(this);
+    this.fetchSchema = this.fetchSchema.bind(this);
+  }
+
+  componentDidMount() {
+    this.setState({ loading: true }, async () => {
```

Comment:
we can set the default `loading` to true, and remove this callback function

Reply:
Loading should be loading when it's actually loading